searchcore/src/vespa/searchcore/proton/metrics/content_proton_metrics.cpp

```diff
@@ -29,6 +29,7 @@ ContentProtonMetrics::ProtonExecutorMetrics::~ProtonExecutorMetrics() = default;
 
 ContentProtonMetrics::ContentProtonMetrics()
     : metrics::MetricSet("content.proton", {}, "Search engine metrics", nullptr),
+      configGeneration("config_generation", {}, "The oldest config generation used by this process", this),
```

Comment:
I suggest renaming this to _config.generation_ in case we need other config related metrics later.

Reply:
Sensible, but we already have this metric.

Reply:
Correction: The existing container metric is actually called "application_generation". 
But since this will have a proton prefix it won't be the same here in any case.

I agree that "config.generation" is better, and perhaps we should move to that both places, and then without any prefix - conceptually this is a config metric, not a proton/container one. It's not a big deal to handle different names though, so not sure it is worth prioritizing.